### ISSUE TITLE
Text un-selectable when embedded in iframe/react-frame-component

### ISSUE THREAD

**Issue by yuxinzhu** [securingsincity/react-ace#839]
tl;dr [Demo of Issue](https://codesandbox.io/s/react-component-frame-ace-editor-grvb0?file=/src/index.js)

I'm attempting to integrate Ace Editor in an iframe via React 16. I am using the react-ace component library, though this issue exists if I implement the editor natively as well.

Although everything mostly works when I wrap the editor in [react-frame-component](https://github.com/ryanseddon/react-frame-component), I cannot select highlight text which makes the editor unusable. I suspect this may have to do with the fact that `react-frame-component` is implemented with Portals, but I'm not sure exactly what the issue is.

I've replicated the issue [here](https://codesandbox.io/s/react-component-frame-ace-editor-grvb0?file=/src/index.js). I don't think this problem exists if I loaded the component with a regular `<iframe src />` from another domain.

PS. I'm using the `renderer.attachToShadowRoot` function since otherwise the styles will be attached globally to the parent DOM.
